how-to/onboarding-guide.md

```diff
@@ -26,37 +26,75 @@ Chief. In the future we hope to find someone with interest in leading peer revie
+
+### The Editorial Team
+It’s helpful to first check with the existing [Editorial Team](https://www.pyopensci.org/about-peer-review/index.html#meet-our-editorial-board), in order to see if anyone can identify reviewers who excelled and may be a good editorial candidate. Do this in a private Slack channel as follows:
+In an ideal world, you’d be able to start your search well in advance of when you think you may need to expand the editorial team. However, due to the influx of new packages, this isn’t always possible. As such, when you begin your search, start with the existing editorial team as well as the pyOpenSci Community Manager.
+
+### The Editorial Team
+It’s helpful to first check with the existing [Editorial Team](https://www.pyopensci.org/about-peer-review/index.html#meet-our-editorial-board), in order to see if anyone can identify reviewers who excelled and may be a good editorial candidate. Do this in a private Slack channel as follows:
+
+* Start a private channel for discussion. This ensures there is not a visible history in a public channel that a new editor may see in the future (this could be awkward for someone to see!)
+* Ping editors using @here to be sure they get a notification, as this is an important topic.
+* Wait for a majority of editors to chime in before inviting someone. Provide editors one week to respond on Slack.
+
+### The pyOpenSci Community Manager
+The Community Manager is available to assist with editor recruitment at any stage in the process, even if it’s before you think you might need editors. Standard recruitment involves targeted social media posts [(similar to this)](https://fosstodon.org/@pyOpenSci/112497485980274296), a blog post, and call-outs in both our monthly and weekly newsletters. To kick off this process, or to request additional ideas for recruitment, reach out to the Community Manager via private message in Slack to let them know:
+
+* That you’re looking for editors!
+* Which type(s) of editors you’re looking for (full, guest).
+* Any specific domains or technical skills you need an editor to have, or if it’s a general call for editors.
+* Any specific packages that need an editor.
+* Any additional promotion you’d like, beyond social media posts, a blog post, and newsletter updates.
+
+```{note}
+pyOpenSci anticipates being able to send targeted emails to community members that meet certain criteria related to domain expertise by the Fall of 2024. Please reach out to the pyOpenSci Community Manager with any questions about this process!
+```
+* Start a private channel for discussion. This ensures there is not a visible history in a public channel that a new editor may see in the future (this could be awkward for someone to see!)
```

Comment:
```suggestion
pyOpenSci anticipates being able to send targeted emails to community members that meet certain criteria related to domain expertise by the Fall of 2024. Please reach out to the pyOpenSci Community Manager with any questions about this process!
```

* Start a private channel for discussion. This ensures there is not a visible history in a public channel that a new editor may see in the future (this could be awkward for someone to see!)
```